types/bull-arena/index.d.ts

```diff
@@ -1,56 +1,61 @@
-// Type definitions for bull-arena 2.6
+// Type definitions for bull-arena 3.0
 // Project: https://github.com/bee-queue/arena/
 // Definitions by: Levi Bostian <https://github.com/levibostian>
+//                 Gaurav Sharma <https://github.com/gtpan77>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 2.8
 
 import { RequestHandler } from "express";
 import { ClientOpts } from "redis";
+import Bull from "bull";
+import Bee from "bee-queue";
```

Comment:
These two dependecies are part of the bull-arena dependencies. So, it is most likely to have these two dependencies.